src/com/view/bar/bar.js

```diff
@@ -145,6 +146,45 @@ export default class ViewBar extends Component {
 		}
 	}
 
+	/**
+	 * Adds the # of notifications in the favicon.
+	 * Shows individual numbers until 9, then 9+ for more.
+	 */
+	refreshFavIconNotificationCount() {
+		const notificationCount = this.state.notifications || 0;
+		// remove existing icon if any
+		const el = document.querySelector("link[rel='shortcut icon']");
+		if (el) el.parentNode.removeChild(el);
+		// reload LD favicon and paint it
+		const canvas = document.createElement('canvas');
+		canvas.setAttribute('width', 64);
```

Comment:
Currently the favicon is 64x64 but it could well be that this is open to change and by that time it could become a bit obscure why it starts to look strange, especially if we would deliver icons based on the device requesting it. So if it is possible it would be better to get the width and height from the `el` above.

Reply:
or even better from the `icon` below.

Reply:
Using icon's size in 71d9966.

However note that the positioning / text size is still hard-coded and will look broken/weird the day that you decide to change the favicon to another size. I tried to play around with different icon sizes (16x16, 32x32 all the way to 256x256) to have something fully generic but it seems like the tradeoff between code readability and genericity wasn't quite worth it here. Unless you foresee that the favicon will change several times in the future, I'd advise to only tackle this specific problem when it presents itself to avoid over-engineering.

However, this is my first time contributing in this codebase so I'll adapt to whatever philosophy you guys have adopted so far.

Thanks for the review!

Reply:
I agree that it's better to keep the code simple, and now the only not looking strange will be the placement and size of the number and that will at least point directly to this feature needing an update.

It's a bit of a pity that the color hex codes need to be duplicerad here, but I don't see any way way too get around that. So the code looks good to me, but it's Mike who finally decides on all

Reply:
I actually originally thought to retrieve those values from the DOM, aka
`window.getComputedStyle(document.querySelector(".view-bar")).backgroundColor`
which works fine but becomes brittle as it adds a hard dependency on the dom structure.

Open to include that change for both colors if you feel that it's better.

Reply:
I think it is better like it is now. Such change would be brittle and quite magic.